tests/pytest_tests/conftest.py

```diff
@@ -26,10 +24,15 @@
 
 
 @pytest.fixture(autouse=True)
-def disable_sentry_in_core(monkeypatch: pytest.MonkeyPatch) -> None:
-    """Disables Sentry for all tests with core."""
+def disable_sentry(monkeypatch: pytest.MonkeyPatch) -> None:
+    """Configures wandb env variables to suitable defaults for tests."""
+    # Don't write to Sentry.
+    monkeypatch.setenv("WANDB_ERROR_REPORTING", "false")
     monkeypatch.setenv("WANDB_CORE_ERROR_REPORTING", "false")
 
+    # Set the _network_buffer setting to 1000. Not sure why. It's undocumented.
```

Comment:
this is to increase the probability of the flow control logic kicking in, if I remember correctly.

Reply:
updated the comment!